tests/static_component/build.cmd

```diff
@@ -0,0 +1,8 @@
+copy "..\target\%2\%1\static_component.lib" ..\..\.windows\%3
+
+if "%3" == "x64" (
```

Comment:
I thought Steve fixed the arch issue?

Reply:
Thanks for the reminder, will pull down the new winmdgenerator and update the PR.